src/pages/EntityDetails/App/App.tsx

```diff
@@ -199,6 +212,23 @@ export default function App(): JSX.Element {
           {tableView === "units" && (
             <>
               <ChipGroup chips={unitChips} descriptor="units" />
+              <div className="entity-details__action-button-row">
+                <Button
+                  appearance="base"
+                  className="entity-details__action-button"
+                  hasIcon={true}
+                  onClick={showActions}
+                  disabled={!enableActionButtonRow}
+                  data-test="run-action-button"
+                >
+                  <img
+                    className="entity-details__action-button-row-icon"
```

Comment:
Could you make this one a proper icon so it can be used like this: `<i className="p-icon--actions"></i>`? Then the icons will be consistent and require less bespoke styling.

Reply:
Also that icon becomes reuseable

Reply:
I tried to do this but because we have multiple base paths that the application runs on we cannot do a simple import in the css. We will have to base64 encode the icon and include it into the css. Would you still like me to do that?

Reply:
https://github.com/canonical-web-and-design/jaas-dashboard/issues/944